rnaget.def

```diff
@@ -0,0 +1,21 @@
+Bootstrap: docker
+From: python:3.7-slim-stretch
+
+%files
+    . /app
+
+%post
+    export HOME="/app"
+    ls /usr/local/lib
+    echo export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.7/site-packages
+    cd /app
+    apt update && apt install -y libhdf5-dev
+    pip3 install --no-cache-dir -r requirements.txt
+    python3 setup.py install
+
+%startscript
+    exec candig_rnaget --port 3000 --logfile log/rnaget.log --database data/rnaget.sqlite
```

Comment:
to the exec, could you add --tmpdata data/tmp/ ?

Reply:
d385342